scripts/spack/configs/mirrors.yaml

```diff
@@ -0,0 +1 @@
+mirrors:: {}
```

Comment:
I was running into a problem where spack's external http mirror wouldnt download to LC. This removes that.